docker-compose.ci.yml

```diff
@@ -28,6 +28,7 @@ services:
                 image_tag: ${IMAGE_TAG}
         image: elifesciences/journal_ci:${IMAGE_TAG}
         volumes:
+            - ./.docker/php.ini:/usr/local/etc/php/conf.d/x-dev.ini
```

Comment:
This has been split into a separate PR: https://github.com/elifesciences/journal/pull/1254

Reply:
Merged.